Tests/test_file_jpeg.py

```diff
@@ -543,10 +543,10 @@ def test_truncated_jpeg_throws_oserror(self) -> None:
     )
     def test_qtables(self, tmp_path: Path) -> None:
```

Comment:
Can we remove the fixture?
```suggestion
    def test_qtables(self) -> None:
```